modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/resources/ExtensionModelResourceFactoryTestCase.java

```diff
@@ -33,15 +38,15 @@ public void before() {
 
   @Override
   protected Class<? extends GeneratedResourceFactory>[] getResourceFactoryTypes() {
-    return new Class[] {ExtensionModelResourceFactory.class};
+    return new Class[] {ExtensionDocumentationResourceGenerator.class};
   }
 
   @Test
   public void generate() throws Exception {
     GeneratedResource resource = resourceFactory.generateResource(extensionModel).get();
-    assertThat(resource.getPath(), equalTo(EXTENSION_MODEL_JSON_FILE_NAME));
-
-    ExtensionModel deserialized = new ExtensionModelJsonSerializer().deserialize(new String(resource.getContent()));
-    assertThat(extensionModel.getName(), equalTo(deserialized.getName()));
+    assertThat(resource.getPath(), equalTo(RESOURCE_NAME));
+    XMLUnit.setIgnoreWhitespace(true);
```

Comment:
there was no test for this before?

Reply:
nope :D, there are for the generation but not for the resource generator using spi